tests/test_output_formatter.cpp

```diff
@@ -1078,6 +1172,8 @@ INSTANTIATE_TEST_SUITE_P(NonOrderableListInfoOutputFormatter, FormatterSuite,
 INSTANTIATE_TEST_SUITE_P(FindOutputFormatter, FormatterSuite, ValuesIn(find_formatter_outputs), print_param_name);
 INSTANTIATE_TEST_SUITE_P(NonOrderableNetworksOutputFormatter, FormatterSuite,
                          ValuesIn(non_orderable_networks_formatter_outputs), print_param_name);
+INSTANTIATE_TEST_SUITE_P(VersionInfoOutputFormatter, FormatterSuite, ValuesIn(version_formatter_outputs),
+                         print_param_name);
```

Comment:
👍

Gotta love parameterized tests :)

Reply:
Populate the test data in arrays and turn the crank. More compact code which is easier to maintain - a win-win 😉 .